### PR TITLE
Make cross platform

### PR DESCRIPTION
**Pull request by aishfenton**
Change checked in bazel file to download-on-first-run the right Bazelisk version, for the platform.

### CODE REVIEW

**Review by ianoc — APPROVED** [review]

File: src/main/scala/afenton/bazel/bsp/BazelBspApp.scala [hunk 1]
```diff
@@ -72,7 +75,8 @@ object BazelBspApp
 
   def main: Opts[IO[ExitCode]] =
     (verboseOpt, verifySetupOpt, setupOpt).mapN { (verbose, verify, setup) =>
-      if setup then writeBspConfig(BuildMetaData.Version).as(ExitCode.Success)
+      if setup then
+        writeBspConfig(Paths.get("").toAbsolutePath).as(ExitCode.Success)
```
Comment (johnynek) [inline src/main/scala/afenton/bazel/bsp/BazelBspApp.scala:79]: nit:

```scala
val currentDir: IO[Path] = IO.blocking(Paths.get("").toAbsolutePath)
```
that is a side effect.

or:
```scala
for {
  cwd <- IO.blocking(Paths.get("").toAbsolutePath)
  _ <- writeBspConfig(cwd)
} yield ExitCode.Success
```
which is maybe a bit more verbose, but I think for-comprehension is very readable for IO heavy code.

File: src/main/scala/afenton/bazel/bsp/BuildMetaData.scala [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package afenton.bazel.bsp
 
 object BuildMetaData {
```
Comment (johnynek) [inline src/main/scala/afenton/bazel/bsp/BuildMetaData.scala:3]: maybe you want to use this:

https://github.com/sbt/sbt-buildinfo

Comment (aishfenton) [reply src/main/scala/afenton/bazel/bsp/BuildMetaData.scala:3]: Will tackle in follow up PR.

**Comment by aishfenton** [comment]
Going to force through for now, as resolving the test issue will require more surgery.

**Pull request merged**